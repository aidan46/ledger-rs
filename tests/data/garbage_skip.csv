type,client,tx,amount
deposit, 1, 1, 46.02
sadgseaa
withdrawal, 1, 1, 4
