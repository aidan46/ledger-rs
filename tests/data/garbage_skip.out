client,available,held,total,locked
1,	44.02	  0,   44.02, false
